addons/mail/static/src/components/discuss/discuss.scss

```diff
@@ -1,96 +1,7 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o-autogrow {
     flex: 1 1 auto;
 }
 
-.o_Discuss {
-    display: flex;
-    height: map-get($sizes, 100);
-    min-height: 0;
```

Comment:
This topic has been mentioned in https://github.com/odoo/odoo/pull/88396

As we can see in the link above, `min-height: 0` is needed whenever `.o_Discuss` sets its containing node as a scrollable flex container and includes a `flex-grow-1` class, in order to make fit its content to height of the container.

Since we don't want to use redundant CSS in this component, `d-flex` + `h-100` makes the sizing work properly, no matter if `flex-grow-1` is declared or not, working without any issues in FF, Chrome and Safari.